holoviews/tests/core/testarchives.py

```diff
@@ -124,3 +124,14 @@ def test_filearchive_json_single_file(self):
             raise AssertionError("No file %r created on export." % fname)
         self.assertEqual(json.load(open(fname, 'r')), data)
         self.assertEqual(archive.listing(), [])
+
+    """
+    A test case that examines the clear() method of
+    FileArchives of io.py
+    """
+    def test_filearchive_clear_file(self):
+        export_name = "archive_for_clear"
+        archive = FileArchive(export_name=export_name, pack=False)
+        archive.add(self.image1)
+        archive.add(self.image2)
+        self.assertEqual(None, FileArchive.clear(archive))
```

Comment:
Not sure what this is testing, shouldn't this be something like:

```suggestion
        archive.clear()
        self.assertEqual(archive._files, {})
```